packages/image_picker/image_picker_ios/lib/image_picker_ios.dart

```diff
@@ -175,6 +175,51 @@ class ImagePickerIOS extends ImagePickerPlatform {
     );
   }
 
+  @override
+  Future<List<XFile>> getMedia({
+    required MediaOptions options,
+  }) async {
+    final MediaSelectionOptions mediaSelectionOptions =
+        _mediaOptionsToMediaSelectionOptions(options);
+
+    final List<String?> paths = await _hostApi.pickMedia(mediaSelectionOptions);
+
+    return paths.map((dynamic path) => XFile(path as String)).toList();
```

Comment:
This shouldn't be `dynamic`, the list is typed. It should be a `String?`, and become `XFile(path!)`.

This could also be inlined with the above, to `return (await ...).map(...).toList().